src/components/App.js

```diff
@@ -35,37 +39,64 @@ function App() {
     setSearch(newSearch)
   }
 
+  const items = { ...localStorage };
+  const videoIds = items ? Object.keys(items) : [];
+  
+  for( var i = 0; i < videoIds.length; i++){   
+    if ( videoIds[i] === 'wa_cert_authenticated') { 
+      videoIds.splice(i, 1); 
+    }
+  }
```

Comment:
If you just want to remove that key 
```suggestion
    const videoIds = items ? Object.keys(items).filter((v) => v !== 'wa_cert_authenticated') : [];
```